pilot/prompts/components/files_list.prompt

```diff
@@ -1,6 +1,8 @@
-{% if files|length > 0 %}Here are files that are currently implemented:
+{% if file_summaries %}These files are currently implemented:{% for fpath, summary in file_summaries.items() %}
```

Comment:
This is intentionally outside `START_OF_FILES` so the summaries don't get replaced.